### PR TITLE
Fix #1879: Convert Fake.IO.FileSystem/Shell.fs to "module based"

### LINKED ISSUE #1879: Convert Fake.IO.FileSystem/Shell.fs to "module based"
### Description

Is someone working on converting [Shell.fs](https://github.com/fsharp/FAKE/blob/master/src/app/Fake.IO.FileSystem/Shell.fs#L16) to "module based" API ?

Converting from `Shell.CopyFile` to `Shell.copyFile`.

If not, I can do it.

### PR DESCRIPTION
**Pull request by MangelMaxime**
@matthid I think we should untrack the `.fake` folder because it's specific to each computer.

### CODE REVIEW

**Comment by MangelMaxime** [comment]
The CI is failing but I think it's coming from the code in `rc_7`branch and not the modifications I made.

**Comment by matthid** [comment]
@MangelMaxime Correct. I'm working on this problem. Seems like some breaking change after `paket update` (from another package)

> I think we should untrack the .fake folder because it's specific to each computer.

We have to include the `intellisense.fsx` file for historical reasons on this repository, because FAKE 4 code needs it (bootstrapping test)

**Comment by matthid** [comment]
@MangelMaxime If you want you can rebase/merge and it should be green again :)

File: build.fsx [hunk 1]
```diff
@@ -828,7 +892,11 @@ Target.create "CheckReleaseSecrets" (fun _ ->
 )
 let executeFPM args =
     printfn "%s %s" "fpm" args
+    #if BOOTSTRAP
+    Shell.exec("fpm", args=args, dir="bin")
```
Comment (matthid) [inline build.fsx]: This still seems to be missing (note that this is actually part of `Fake.Core.Process`)

Comment (MangelMaxime) [reply build.fsx]: Ah yes, I just replace all `Shell.PascalCase` to `Shell.camelCase` this one was too much :)

Let's see if the CI is ok now

**Comment by matthid** [comment]
Ok lets see, thanks!

**Pull request merged**